internal/testutil/testdata/updateMergeConflict/Kptfile

```diff
@@ -1,4 +1,4 @@
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1alpha2
```

Comment:
Similar comment. The schema has changed. There's no `upstream.git.commit` any more.

@mortent Please review the use of these testdata.

Reply:
I want to make sure we understand the scope of what is deleted.  helloworld-set folder is deleted but it seems like there are no tests failing.  That leads me to believe that this test data was not used in any of the current tests, just something that needed to be cleaned up for a while.

We might argue that whenever we deleted the test that was using this test data that we might have lost some coverage, but that's not related to this change.